### DISCUSSION TITLE
Homepage example theorem

### DISCUSSION THREAD

**Discussion by StevenClontz** [pi-base/data#549] [Ideas]
Our website currently uses first-countable and locally pseudometrizable as our example theorem/counterexamples:

https://topology.pi-base.org/

I've changed this to CCC + separable in a [pending pull request](https://github.com/pi-base/web/pull/132):

https://ec3173a4.topology.pages.dev/ 

I think these are more natural properties that visitors will be familiar with. Thoughts?

**Comment by marswill** [comment]
As someone who was pretty unfamiliar with general topology beyond what a grad student knows until quite recently, I think maybe a better choice might be ~"separable but not first countable."~

"Separable + first countable but not second countable"

**Reply by marswill** [reply]
Actually, am I misremembering, or did that use to actually be the example?

**Reply by StevenClontz** [reply]
Current example is "first countable, not locally pseudometrizable"; it used to be "first countable, not metrizable" before the theorem was generalized.

I agree "Separable + first countable but not second countable" is an interesting non-theorem, but it's not captured as the converse of a single existing theorem.

**Reply by marswill** [reply]
Ah, I see.  And I guess I was conflating it with the non-theorem advertised on the explore page, which is indeed still there.

Up until a few days ago, we could have done the converse to "second countable implies separable", which are both terms any graduate student should know, except now it would have to be "second countable implies hereditarily separable", which a grad student should still be able to infer the meaning of, but isn't quite as elegant.

I think this hints at a larger problem the site might have going forward, as theorems get refined to be sharper and sharper, but correspondingly more and more technical.  Unfortunately every idea I've had about how to address that would likely entail reasonably substantial changes to the code base.  

For now, if the criterion is "something a grad student who isn't yet a specialist in general topology ought to be able to appreciate", then
" $T_2$ implies has closed retracts"  ([T310](https://topology.pi-base.org/theorems/T000310)) is sort of nice because "has closed retracts" is exactly what it says on the tin, and grad students should know what retracts are.

Other possibilities which might be a little too basic might be [T39](https://topology.pi-base.org/theorems/T000039) or [T40](https://topology.pi-base.org/theorems/T000040),  "path connected implies connected," and "arc connected implies path connected"

The counterexamples to the converses are sort of nifty, but the theorems themselves are bordering on tautologies.

**Reply by StevenClontz** [reply]
I actually haven't dealt with retracts much until the past year digging through T1-not-T2 stuff. But I really like T40 "path connected implies connected" as a possibility.

I'm adding this to our agenda for our meeting this week.
